### PR TITLE
Add `rateLimitGroup` field to GraphQL logs

### PR DESCRIPTION
**Pull request by kamilatwork**
This is a stopgap solution while waiting for Entur to properly annotate rate limiting groups.

### CODE REVIEW

File: src/graphql/graphql-client.ts [hunk 1]
```diff
@@ -66,6 +66,8 @@ function createClient(url: string) {
     const loggingLink = new ApolloLink((operation, forward) => {
       return forward(operation).map(response => {
         const context = operation.getContext();
+        const operationNameGroup =
+          operation.operationName == 'Trips' ? 'trips' : 'nontrip';
```
Comment (rosvik) [inline src/graphql/graphql-client.ts:70]: Just curious: Where does `operationName` field come from? Can't find any references to it other places in the BFF.

Comment (kamilatwork) [reply src/graphql/graphql-client.ts:70]: My understanding is that this is something that apollo clients sets, but not sure where it's coming from. For example: `bff/v1/departures-realtime` endpoint returns `GetDepartureRealtime` as `operationName`.

Comment (rosvik) [reply src/graphql/graphql-client.ts:70]: Aha, it's probably the name of the GraphQL query then 💡 E.g. https://github.com/AtB-AS/atb-bff/blob/main/src/service/impl/trips/journey-gql/trip.graphql#L1:L1 and https://github.com/AtB-AS/atb-bff/blob/main/src/service/impl/realtime/journey-gql/departure-time.graphql#L1:L1

Good to know, as changing query names could have some side effects now.

Comment (mjenssen) [reply src/graphql/graphql-client.ts:70]: Yes, it's the (optional) name you give a query in GraphQL. Ideally, we would like to have this rate limit group returned from Entur, but this is a (hopefully) temporary workaround.

**Pull request merged**